lib/modules/manager/terraform/extract.spec.ts

```diff
@@ -17,6 +17,8 @@ const helm = loadFixture('helm.tf');
 const lockedVersion = loadFixture('lockedVersion.tf');
 const lockedVersionLockfile = loadFixture('rangeStrategy.hcl');
 const terraformBlock = loadFixture('terraformBlock.tf');
+const tfeWorkspaceBlock = loadFixture('tfeWorkspaceTerraformVersion.tf');
+const tfeWorkspaceNoVersion = loadFixture('tfeWorkspaceNoTerraformVersion.tf');
```

Comment:
Combine this two file, to test the actual code.   
Currently it is aborting during the pre-flight check. https://github.com/renovatebot/renovate/pull/14522/files#diff-5ab62e69287f1f4f2c26b5d60412284ac7ca35b03655fdd19a126da08d7ba944R49

Reply:
That makes so much sense, thanks for pointing it out. That actually also caught a bug.